RecyclerViewKotlin/README.md

```diff
@@ -1,57 +1,5 @@
+#RecyclerView Kotlin
```

Comment:
The previous README is much more detailed, can you copy over it's content and update it?